components/home/Splash.tsx

```diff
@@ -6,12 +6,17 @@ const Splash = () => (
       src="/realfagsbygget.png"
       alt="Realfagsbygget"
       // unsure what the height and width should be
-      height={100000}
```

Comment:
Bra du bytta, men hvorfor var den 100000?

Reply:
Visste ikke hva den skulle stå på før, så tok bare 100000 for å få best mulig oppløsning